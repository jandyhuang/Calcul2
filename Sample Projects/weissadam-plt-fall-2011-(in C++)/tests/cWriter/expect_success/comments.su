function main[] returns int{
	/* print("nothing"); */
	/*  
	print ("nothing");
	*/  
	print("something");
	print("this are comment delimiters: /* comment */");
	return 0;
	}
