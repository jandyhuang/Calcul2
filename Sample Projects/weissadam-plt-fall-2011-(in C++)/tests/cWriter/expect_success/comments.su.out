something
this are comment delimiters: /* comment */
